### ISSUE TITLE
Distractors implemented as separate threads

### ISSUE THREAD

**Issue by breznak** [breznak/openDS#61]
- [ ] add a `duration` field to distractors
- [ ] make them runable `threads` and show it for the duration
  - basically the thread would: `D.spawn(); T.sleep(duration); D.remove()`
  - called from `D.update()`

This will make them more independent of the sim control loop.

**Comment by breznak** [comment]
@JohnnyMarek how is the "disappearing" handled now? is it the `Timer` and magic constants `5, 15` which I didn't understand? In sim loop?

**Comment by JohnnyMarek** [comment]
yes, 5 sec till anything spawns, 15 secs it stays unles it is text or collect (collect is removed when collected or after 15 sec [failed attempt]. text stays till resolved
